### PR TITLE
Trivial: Properly comment about shutdown process in init.cpp file.

### PR DESCRIPTION
**Pull request by wraith7**

### CODE REVIEW

**Comment by jonasschnelli** [comment]
ACK d7f8a6f1799076125827749ea61f0abbce201e7d
ultra nit: start a sentence with "And then"

File: src/init.cpp [hunk 1]
```diff
@@ -102,12 +102,11 @@ static const char* FEE_ESTIMATES_FILENAME="fee_estimates.dat";
...
-// signal handler sets fRequestShutdown, which triggers
-// the DetectShutdownThread(), which interrupts the main thread group.
-// DetectShutdownThread() then exits, which causes AppInit() to
-// continue (it .joins the shutdown thread).
-// Shutdown() is then
-// called to clean up database connections, and stop other
+// signal handler sets fRequestShutdown, which makes main thread's
+// WaitForShutdown() interrupts the thread group.
+// And then, WaitForShutdown() makes all other on-going threads
+// in the thread group join the main therad.
```
Comment (MeshCollider) [inline src/init.cpp]: nit: typo in thread

**Comment by fanquake** [comment]
Please squash into a single commit.

**Comment by wraith7** [comment]
Squashed

**Pull request merged**